src/handleAction.js

```diff
@@ -2,19 +2,31 @@ function isFunction(val) {
   return typeof val === 'function';
```

Comment:
Sorry, https://github.com/acdlite/redux-actions/pull/97#discussion_r70165690 wasn't meant to suggest this change - it's more of a longer-term vision for this library. You can revert this commit if you like.

(Also, you can use ES6 string interpolation below.)